templates/apiary-hms-readonly.json

```diff
@@ -26,11 +26,11 @@
     ],
     "environment":[
      {
-        "name": "dbhost",
+        "name": "MYSQL_DB_HOST",
         "value": "${db_host}"
```

Comment:
The values don't need to change?

Reply:
values are just internal references, no that imporant, but will update anyway.